### ISSUE TITLE
The starter tutorial cannot run well. AttributeError: `load_load` function do not exist.

### ISSUE THREAD

**Issue by yhzhu99** [WenjieDu/PyPOTS#155]
### 1. System Info

Latest pypots (v0.1.1)
python 3.10
Debian 11

### 2. Information

- [X] The official example scripts
- [ ] My own created scripts

### 3. Reproduction

Simply run the [Quick-start Examples](https://docs.pypots.com/en/latest/examples.html#)

### 4. Expected behavior

Meet error:

```
Traceback (most recent call last):
  File "/home/zhuyh/projects/ImputeEHR/pypots.py", line 46, in <module>
    saits.load_load("examples/saits/manually_saved_saits_model")
AttributeError: 'SAITS' object has no attribute 'load_load'
```

**Comment by WenjieDu** [comment]
Hi there 👋,

Thank you so much for your attention to PyPOTS! If you find PyPOTS helpful to your work, please star⭐️ this repository. Your star is your recognition, which can help more people notice PyPOTS and grow PyPOTS community. It matters and is definitely a kind of contribution to the community.

I have received your message and will respond ASAP. Thank you for your patience! 😃

Best,
Wenjie

**Comment by WenjieDu** [comment]
Sorry, there is a typo in this line of code. Please run `saits.load_model("examples/saits/manually_saved_saits_model")`

**Comment by WenjieDu** [comment]
The typo has been fixed in PR#156. I'm closing this issue. If you use SAITS and like it, please consider star 🌟 the SAITS repo https://github.com/WenjieDu/SAITS/ to make more people notice this work. Thanks. 😊
